sql/V7__AuthToken.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE authtokens (
+  token VARCHAR
```

Comment:
Can you make these columns `NOT NULL`?

Reply:
Oh man. I should have caught that. Just pushed new commit.